package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "business-filings-ui",
-  "version": "5.8.6",
+  "version": "5.9.0",
```

Comment:
This is the start of a new feature so it makes sense to update the minor release number.